docs/main_components.md

```diff
@@ -15,6 +15,16 @@ Python clients, and the Javascript-based web user interface and configuration.
 
 The server layer serves the purpose of authentication & provides a common API
 for interaction with clients & other external application, as also the Web UI.
+It is a passive component which receives incoming queries and forward them to 
+the core. For any general user who wants to access the Rucio WebUI, one must 
+fork the WebUI repository (https://github.com/rucio/webui) and after follwing
```

Comment:
Yeah, this is not really right either :-) The WebUI is deployed centrally (once), and users access that. No need for users to install it themselves.